ui/pages/ProjectParametersPage.qml

```diff
@@ -0,0 +1,13 @@
+import QtQuick 2.12
```

Comment:
you should move Project Settings inside the global ParametersPage to provide a better UX.
It will be a big ParametersPage for now but a redesign will need to be discussed and implemented for parameters.

Reply:
With the merge to ParametersPage, this page needs to be removed.